scripts/main/sidebar.js

```diff
@@ -117,6 +117,15 @@ sidebar.changeAttr = function(attr, value = '-', dangerouslySetInnerHTML = false
 
 };
 
+sidebar.secondsToHMS = function(d) {
+	d = Number(d);
+	var h = Math.floor(d / 3600);
+	var m = Math.floor(d % 3600 / 60);
+	var s = Math.floor(d % 3600 % 60);
+
+	return ((h > 0) ? h + 'h' : '') + ((m > 0) ? m + 'm' : '') + ((s > 0) ? s + 's' : '');
```

Comment:
I would use
`.toString()` to make sure of the conversion of `h`, `m` and `s`

Reply:
Done. I also fixed a case of sub-0s video where it should still print "0s" rather than nothing.